.gitignore

```diff
@@ -37,3 +37,6 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+
+# Secrets
+.npmrc
```

Comment:
I added a `.npmrc` with a secret npm key for this project so we won't have to change the global npm key.

Reply:
key should not be part of the project FYI.